torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -0,0 +1,65 @@
+#include <torch/csrc/jit/passes/normalize_ops.h>
+#include <c10/util/Exception.h>
+
+namespace torch {
+namespace jit {
+
+// map from op alias -> normalized op
+static const std::unordered_map<Symbol, Symbol> alias_map = {
+    {aten::absolute, aten::abs},
+    {aten::absolute_, aten::abs_},
+};
+
+void replaceNodeWithNewSymbol(Node* node, Symbol new_symbol) {
```

Comment:
Would this be a useful method to expose on `Node` or `Graph`? I can definitely think of places I've wanted a function like this elsewhere

Reply:
Do you have any examples of other call sites ?